demos/projects/ESPRESSIF/esp32/components/azure-iot-middleware-freertos/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(ROOT_PATH
+    ${CMAKE_CURRENT_LIST_DIR}/../../../../../..
+)
+
+set(AZURE_IOT_MIDDLEWARE_FREERTOS 
+    ${ROOT_PATH}/libs/azure-iot-middleware-freertos
+)
+
+file(GLOB_RECURSE COMPONENT_SOURCES
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/source/*.c  
+)
+
+list(APPEND COMPONENT_SOURCES
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/ports/coreMQTT/azure_iot_core_mqtt.c
+)
+
+idf_component_get_property(FREERTOS_DIR freertos COMPONENT_DIR)
+
+set(COMPONENT_INCLUDE_DIRS
+    ${FREERTOS_DIR}/include/freertos
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/.github/config
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/source/include
```

Comment:
config are generally provided by application. Config under .github/config is mostly for testing.

Reply:
Agreed here, config should either be in KConfig or a config folder in project.

Reply:
Sure, removed.